specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2025-02-01/vaults.json

```diff
@@ -0,0 +1,2369 @@
+      "properties": {
+        "crossSubscriptionRestoreState": {
+          "enum": [
+            "Enabled",
+            "Disabled",
+            "PermanentlyDisabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "crossSubscriptionRestoreState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "RestoreSettings": {
+      "description": "Restore Settings  of the vault",
+      "type": "object",
+      "properties": {
+        "crossSubscriptionRestoreSettings": {
+          "description": "Settings for CrossSubscriptionRestore",
+          "$ref": "#/definitions/CrossSubscriptionRestoreSettings"
+        }
+      }
+    },
+    "ImmutabilitySettings": {
+      "description": "Immutability Settings of vault",
+      "type": "object",
+      "properties": {
+        "state": {
+          "enum": [
+            "Disabled",
+            "Unlocked",
+            "Locked"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ImmutabilityState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SoftDeleteSettings": {
+      "description": "Soft delete Settings of vault",
+      "type": "object",
+      "properties": {
+        "softDeleteState": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled",
+            "AlwaysON"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SoftDeleteState",
+            "modelAsString": true
+          }
+        },
+        "softDeleteRetentionPeriodInDays": {
+          "format": "int32",
+          "description": "Soft delete retention period in days",
+          "type": "integer"
+        },
+        "enhancedSecurityState": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled",
+            "AlwaysON"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "EnhancedSecurityState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SourceScanConfiguration": {
+      "description": "Source scan configuration of vault",
+      "type": "object",
+      "properties": {
+        "state": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "State",
+            "modelAsString": true
+          }
+        },
+        "sourceScanIdentity": {
+          "$ref": "#/definitions/SourceScanIdentity"
+        }
+      }
+    },
+    "SourceScanIdentity": {
+      "type": "object",
+      "description": "Identity details to be used for source scan operation",
+      "properties": {
+        "identityType": {
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned"
+          ],
+          "type": "string",
+          "x-ms-enum": {
```

Comment:
>"x-ms-enum" [](http://example.com/codeflow?start=10&length=11)

Not blocking ARM signoff, but adding descriptions for each enum value is recommended before merging.